_tools/release_docs/api-reference-process.go

```diff
@@ -67,7 +67,7 @@ func stripHTML(filename string) error {
 // replace http://releases.k8s.io/HEAD/docs/xxx.md with
 // http://kubernetes.io/v1.0/docs/xxx.html
 func fixLinks(fileName, outputDir string) error {
-	mdRE := regexp.MustCompile(`http://releases.k8s.io/HEAD/docs(.*?\.)md`)
```

Comment:
The case sounds quite hypothetical. Why will we refer a v1.0 doc in a v1.1 definitions.html? Anyway, I guess it's no big deal. I'll make the change and fix versionize-docs if there are further problems.